react-app/src/components/AppSideBar/Footer.tsx

```diff
@@ -0,0 +1,106 @@
+import React from "react";
+import cn from "classnames";
+import Divider from "../common/Divider/Divider";
+import AppButton from "../common/Buttons/AppButton";
+
+const Footer: React.FC = () => {
+  return (
+    <div className={cn("mx-11", "mt-9", "sm:mt-8", "mb-16")}>
+      <Divider />
+      <div
+        className={cn(
+          "mt-8",
+          "text-center",
+          "text-sm",
+          "leading-5",
+          "font-normal",
+          "text-gray-500",
+          "sm:flex",
+          "sm:items-center",
+          "sm:mt-5"
+        )}
+      >
+        <p className={cn("sm:grow", "sm:text-left")}>© 2022 Likecoin</p>
+        <p className={cn("mt-6", "sm:mt-0")}>Get Token</p>
+        <div
+          className={cn(
+            "flex",
+            "justify-center",
+            "items-center",
+            "my-3",
+            "sm:px-3"
+          )}
+        >
+          <AppButton
+            messageID="footer.osmosis"
+            theme="secondary"
+            size="small"
+            type="anchor"
+            href="https://app.osmosis.zone/?from=UST&to=LIKE"
+            className={cn(
+              "text-sm",
+              "leading-4",
+              "font-medium",
+              "text-likecoin-darkgrey",
+              "m-0.5"
+            )}
+          />
+          <AppButton
+            messageID="footer.liquid"
+            theme="secondary"
+            size="small"
+            type="anchor"
+            href="https://app.liquid.com/exchange/LIKEUSDT"
```

Comment:
can we get these links from config.js? just so we don't need tor redeploy the entire site if we want to update the url

Reply:
updated 🙇‍♂️